src/processors/file-handler.ts

```diff
@@ -47,25 +50,25 @@ const JSON_LOADER_OPTIONS = {
   ]
 };
 
-export function isGeoJson(json) {
+export function isGeoJson(json: any): boolean {
   // json can be feature collection
   // or single feature
   return isPlainObject(json) && (isFeature(json) || isFeatureCollection(json));
 }
 
-export function isFeature(json) {
+export function isFeature(json: any): boolean {
```

Comment:
` isFeature(json: unknown): json is Feature`

Reply:
fixed